assets/src/edit-story/components/form/media.js

```diff
@@ -193,6 +194,14 @@ function MediaInput({
 
   const isMenuVisible = isHovering || isFocused;
 
+  const buttonRef = useRef();
+
+  useEffect(() => {
```

Comment:
Can you add some comments about what's going on here? Why can't this just be passed through? Wouldn't `isFocused` just pass through normally?

Reply:
`isFocused` was used to determine `isMenuVisible`, the button itself was not focused before without the ref